### PR TITLE
Usability Improvements

### PR DESCRIPTION
**Pull request by damyon**
Changes included in this patch are:

* Json -> HTML viewer
* Relative URLs fixed
* Change from a collapsible list to using tabs for the Pages/Forms/Documents/JSON/Images
* Ask for a "prefix" to add before each client key is generated (so you can tell which client the key is for)

To test the performance, I generated a json file that was 38MB and had 15000 pages. It took a few seconds to load the site, but once it was loaded it was fine. I tried another one that was 129M and it needed too much memory from the browser :(

### CODE REVIEW

**Comment by damyon** [comment]
I added Scoring for each page of content based on 

https://pdfs.semanticscholar.org/88a8/db54bc099ce7ca91bcfd451b9b6627e80e73.pdf

I also made some better "action urls" for forms and used a copy to clip board button for the forms and pages instead of using text areas.

**Comment by damyon** [comment]
This master branch has been updated with all the latest changes.

File: public/persist.php [hunk 1]
```diff
@@ -136,6 +211,134 @@ function perform_request_remove($args, $indexJson) {
...
+  // Add this to the pending list.
+  $client = (object) $args;
+
+  $client->clientid = strtolower(preg_replace("/[^A-Za-z0-9 ]/", '', $client->company));
+  $client->key = $client->clientid . '-' . generate_random_string(6);
+  array_push($currentVerify, $client);
+  file_put_contents($verifyFile, json_encode($currentVerify));
+
+  // Send them an email about it.
+  $subject = 'Access to Website Crawler: ' . $client->company;
```
Comment (jez500) [inline public/persist.php]: User entered values should probably be sanitised `htmlentities($client->company)`

File: public/persist.php [hunk 2]
```diff
@@ -136,6 +211,134 @@ function perform_request_remove($args, $indexJson) {
...
+}
+
+/**
+ * Parse the post params for the new client contact.
+ *
+ * @return array
+ */
+function parse_request_new_client() {
+  $params = [];
+  $params['firstName'] = $_POST['firstName'];
```
Comment (jez500) [inline public/persist.php]: As above, should be sanitised, never trust a $_POST value

**Pull request merged**